<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Jayarees-MacBook-Pro.local" name="Stepdef.popbitch_transaction" tests="7" failures="1" timestamp="19 Aug 2019 09:08:10 GMT" time="119.219" errors="0">
  <testcase name="i_am_a_registered_Agate_User_and_I_login_in_to_popbitch_staging_with_username_as_and_password_as" time="52.220" classname="Stepdef.popbitch_transaction"/>
  <testcase name="i_read_a_premium_article" time="12.976" classname="Stepdef.popbitch_transaction"/>
  <testcase name="the_wallet_balance_and_free_point_are_deducted_by_p" time="10.144" classname="Stepdef.popbitch_transaction"/>
  <testcase name="the_New_European_wallet_balance_is_updated_same_as_popbitch_and_free_point_remains_uneffected" time="0.001" classname="Stepdef.popbitch_transaction"/>
  <testcase name="reactions_wallet_balance_is_updated_and_free_point_remains_uneffected" time="14.473" classname="Stepdef.popbitch_transaction"/>
  <testcase name="Cornwall_wallet_balance_is_updated_and_free_point_remains_uneffected" time="14.199" classname="Stepdef.popbitch_transaction">
    <failure type="java.lang.AssertionError" message="expected:&lt;1.00&gt; but was:&lt;60&gt;">
      <![CDATA[java.lang.AssertionError: expected:<1.00> but was:<60>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
at Stepdef.popbitch_transaction.Cornwall_wallet_balance_is_updated_and_free_point_remains_uneffected(popbitch_transaction.java:319)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Cornwall_wallet_balance_is_updated_and_free_point_remains_uneffected -->
  <testcase name="cricketer_wallet_balance_remains_and_is_uneffected" time="15.206" classname="Stepdef.popbitch_transaction"/>
</testsuite> <!-- Stepdef.popbitch_transaction -->
